README.md

```diff
@@ -19,6 +26,7 @@ Learn how to engage with the Kubernetes community on the [community page](http:/
 You can reach the maintainers of this project at:
 
 - [Slack](http://slack.k8s.io/)
+  * sig-storage
```

Comment:
Might also want to put wg-csi since I'm sure they'll be interested.

Reply:
Good call, will do.